### ISSUE TITLE
paper-toggle-button accessibility issues 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-toggle-button#11]
Two accessibility issues found: 

1) The contrast of the gray circles prior to being clicked / activated is too poor. It's very difficult to see those on the page. Can we boost the contrast of that gray?

2) There is a bit of inconsistency across screen readers for spoken feedback about the state of the toggle. 

On Windows, across Chrome and Firefox, NVDA will read out that the toggle button is "pressed" but does not speak when it is unpressed. This could be a slightly confusing user experience without spoken confirmation. 

On Chrome with VoiceOver on the Mac, I can only hear the state of the toggle when I use the force click command: VO + space to toggle it. I cannot however hear whether or not the toggle is pressed or not pressed when I use enter to toggle it. This is not a P1 issue by any means, but it is an inconsistency.

**Comment by notwaldorf** [comment]
Re: reading out the state when it changes. I think "not reading" is consistent with a native radio button/checkbox (unless i'm misunderstanding how to repro this). It seems that if I start off with an unselected checkbox (VO reads it as `unselected`), and press change to toggle its state, VO doesn't re-read it as `selected.

**Comment by lpalmaro** [comment]
So, on ChromeOS with ChromeVox, I can either use space or enter to
toggle/untoggle, and I get feedback for both "pressed" and "unpressed"
(which is smooth).

With VO on Mac, I can press control option space to force click to toggle,
and I hear "select toggle button". If I press again, I hear "deselect
toggle button", which is helpful. This doesn't occur when pressing enter to
toggle it though – when pressing enter to toggle, I hear no spoken
feedback.

On Windows / Chrome / NVDA, I can either press enter or space to toggle the
radio button, and when I do that, I hear "pressed", but never "unpressed".
Just seems like a bit of inconsistency.
- Dominic -- is this expected/okay behavior across NVDA / VO?

**Comment by lpalmaro** [comment]
It's probably best to compare it to a similar native control. Try this url
for a collection of pure-HTML native controls. If the Polymer control is
significantly worse than the native control with the same screen reader,
that's potentially a bug.

http://goo.gl/P4qRhp

If you think it's coded correctly and can't figure out why the feedback is
wrong, assign it to me. That's something I can definitely help with.

**Comment by notwaldorf** [comment]
Closing this as per offline discussion (which was: none of the native elements announce anything on change)
